e2e/various-suite/keybinds.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { e2e } from '../utils';
+import { fromBaseUrl } from '../utils/support/url';
+
+const options = {
+  defaultCommandTimeout: 5 * 1000,
+};
```

Comment:
this also isn't necessary for me locally

Reply:
to be fair (either way), this reduces the maximum timeout to get failures sooner. By default i think it waits like 30 seconds for elements?

Reply:
ah fair 👍